examples/BUILD

```diff
@@ -33,7 +34,6 @@ filegroup(
     name = "contrib_configs",
     srcs = glob(
         [
-            "golang/*.yaml",
```

Comment:
i think this should not be removed - it needs to be updated and the additional config added